app/src/main/java/de/tum/in/tumcampusapp/component/ui/onboarding/StartupActivity.java

```diff
@@ -88,37 +87,6 @@ private void init() {
         ImplicitCounter.count(this);
         ImplicitCounter.submitCounter(this);
 
-        // For compatibility reasons: big update happened with version 35
-        int prevVersion = Utils.getSettingInt(this, Const.APP_VERSION, 35);
-
-        // get current app version
-        int currentVersion = Utils.getAppVersion(this);
-        boolean newVersion = prevVersion < currentVersion;
-        if (newVersion) {
-            this.setupNewVersion();
-            Utils.setSetting(this, Const.APP_VERSION, currentVersion);
-        }
```

Comment:
what happened to `setupNewVersion()`?
I think we can either completely remove it then?

Reply:
I'm not really sure, we probably can. What do you think @kordianbruck ?

Reply:
We can probably get rid of that as well. Just a question of if we already want to skip resetting the DB. I had to trigger a destructive migration for the current 1.5.6 release, as we were getting many crash reports from users.